tests/internal/import_hooks/test_integration.py

```diff
@@ -48,3 +50,67 @@ def test_import_hooks(hooks):
         # DEV: Slightly redundant to check twice, but good to be sure
         module_hook.assert_called_once_with(urllib)
         module_hook.assert_called_once_with(sys.modules[mod_name])
+
+
+@run_in_subprocess
+class ImportHookTestCase(SubprocessTestCase):
+    def setUp(self):
+        # Reset the hooks before each test case
+        import_hooks.hooks.reset()
```

Comment:
if we always run these in a subprocess do we need to reset them?

Reply:
We do not! Good catch 😄